### PR TITLE
Added publications for UVL paper

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342402},
+  doi = {10.1145/3307630.3342402},
+  booktitle = MODEVAR,
+  pages = {137–142},
+  numpages = {6},
+  keywords = {variability, software product lines, feature models},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1279]: s. a.

File: literature.bib [hunk 2]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  author = {Kamali, Seiede Reyhane and Kasaei, Shirin and Lopez-Herrejon, Roberto E.},
+  title = {{Answering the Call of the Wild? Thoughts on the Elusive Quest for Ecological Validity in Variability Modeling}},
+  year = 2019,
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342400},
+  doi = {10.1145/3307630.3342400},
+  booktitle = MODEVAR,
+  pages = {143–150},
```
Comment (pmbittner) [inline literature.bib:1292]: --

File: literature.bib [hunk 3]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  numpages = {8},
+  keywords = {feature models, open source projects, variability models},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1296]: s.a.

File: literature.bib [hunk 4]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342401},
+  doi = {10.1145/3307630.3342401},
+  booktitle = MODEVAR,
+  pages = {162–169},
+  numpages = {8},
+  keywords = {variability language, domain specific language, variability specification},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1313]: s.a.

File: literature.bib [hunk 5]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  author = {Villota, Angela and Mazo, Ra\'{u}l and Salinesi, Camille},
+  title = {{The High-Level Variability Language: An Ontological Approach}},
+  year = 2019,
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342401},
+  doi = {10.1145/3307630.3342401},
+  booktitle = MODEVAR,
+  pages = {162–169},
```
Comment (pmbittner) [inline literature.bib:1309]: --

File: literature.bib [hunk 6]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342406},
+  doi = {10.1145/3307630.3342406},
+  booktitle = MODEVAR,
+  pages = {182},
+  numpages = {1},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1329]: s.a.

File: literature.bib [hunk 7]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  author = {Galindo, Jos\'{e} A. and Benavides, David},
+  title = {{Towards a New Repository for Feature Model Exchange}},
+  year = 2019,
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342405},
+  doi = {10.1145/3307630.3342405},
+  booktitle = MODEVAR,
+  pages = {170–173},
```
Comment (pmbittner) [inline literature.bib:1342]: --

File: literature.bib [hunk 8]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  numpages = {4},
+  keywords = {variability, requirements, feature model repository, characteristics},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1346]: .

File: literature.bib [hunk 9]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342398},
+  doi = {10.1145/3307630.3342398},
+  booktitle = MODEVAR,
+  pages = {151–157},
+  numpages = {7},
+  keywords = {textual specification languages, software product lines, variability modeling},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1363]: .

File: literature.bib [hunk 10]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  author = {Beek, Maurice H. ter and Schmid, Klaus and Eichelberger, Holger},
+  title = {{Textual Variability Modeling Languages: An Overview and Considerations}},
+  year = 2019,
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342398},
+  doi = {10.1145/3307630.3342398},
+  booktitle = MODEVAR,
+  pages = {151–157},
```
Comment (pmbittner) [inline literature.bib:1359]: --

File: literature.bib [hunk 11]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  author = {Rabiser, Rick},
+  title = {{Feature Modeling vs. Decision Modeling: History, Comparison and Perspectives}},
+  year = 2019,
+  isbn = {9781450366687},
+  publisher = ACM,
+  address = NY,
+  url = {https://doi.org/10.1145/3307630.3342399},
+  doi = {10.1145/3307630.3342399},
+  booktitle = MODEVAR,
+  pages = {134–136},
```
Comment (pmbittner) [inline literature.bib:1377]: -- (I won't continue making this comment. Please check the other occurences.

File: literature.bib [hunk 12]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  numpages = {3},
+  keywords = {variability modeling, decision modeling, feature modeling},
+  location = {Paris, France},
+  series = {SPLC '19}
```
Comment (pmbittner) [inline literature.bib:1381]: (same here)

File: literature.bib [hunk 13]
```diff
@@ -1451,6 +1661,30 @@ @article{LNTS:JSS19
...
+  title={{PyFml-a Textual Language For Feature Modeling}},
+  author={Al-Azzawi, Ali Fouad},
+  booktitle = IJSEA,
+  year=2018
+}
+
+
+
+
+
```
Comment (pmbittner) [inline literature.bib:1675]: too much whitespace

File: literature.bib [hunk 14]
```diff
@@ -1451,6 +1661,30 @@ @article{LNTS:JSS19
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2018 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% could not find additional information
+@article{A:IJSEA18,
+  title={{PyFml-a Textual Language For Feature Modeling}},
+  author={Al-Azzawi, Ali Fouad},
+  booktitle = IJSEA,
+  year=2018
```
Comment (pmbittner) [inline literature.bib:1669]: publisher, address missing

File: literature.bib [hunk 15]
```diff
@@ -1451,6 +1661,30 @@ @article{LNTS:JSS19
...
+  author={Schmid, Klaus and Kr{\"o}her, Christian and El-Sharkawy, Sascha},
+  booktitle= SPLC,
+  publisher = ACM,
+  address = NY,
+  pages={306--306},
+  year=2018
+}
+
+
+
```
Comment (pmbittner) [inline literature.bib:1687]: too much whitespace

File: literature.bib [hunk 16]
```diff
@@ -4199,6 +4445,16 @@ @article{RTS+:JLAMP16
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2015 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{SH15,
+  title={{Variability Exchange Language-A Generic Exchange Format for Variability Data}},
+  author={Schulze, Michael and Hellebrand, Robert},
+  booktitle={Software Engineering (Workshops)},
+  pages={71--80},
+  year= 2015
+}
```
Comment (pmbittner) [inline literature.bib:4454]: Is this a `@book`?

File: literature.bib [hunk 17]
```diff
@@ -6439,6 +6695,15 @@ @article{WG+:JSS14
 
 }
 
+% Could not find additional information
+@article{KKP+14,
```
Comment (pmbittner) [inline literature.bib:6699]: KKP+:CoRR14

File: literature.bib [hunk 18]
```diff
@@ -6635,21 +6900,19 @@ @article{MH:JSS13
...
-  booktitle = ICCS,
-  location = {Baton Rouge, LA, USA},
-  series = LNCS,
-  volume = {5544},
-  pages = {367--376},
-  publisher = Springer,
-  address = BerlinHeidelberg,
-  doi = {10.1007/978-3-642-01970-8\_36},
-  year = 2009,
-  kc-tags = {BuDDy, CUDD, BDD Evaluation}
```
Comment (pmbittner) [inline literature.bib:6650]: Why is this deleted? Or is this a diff anomaly?

File: literature.bib [hunk 19]
```diff
@@ -6635,21 +6900,19 @@ @article{MH:JSS13
...
+@article{GWT+:TSE13,
+  title= {{Variability in Software Systems-a Systematic Literature Review}},
+  author={Galster, Matthias and Weyns, Danny and Tofan, Dan and Michalik, Bartosz and Avgeriou, Paris},
+  journal= TSE,
+  volume={40},
+  number={3},
+  pages={282--306},
+  year= 2013
```
Comment (pmbittner) [inline literature.bib:6910]: publisher missing

File: literature.bib [hunk 20]
```diff
@@ -8181,6 +8444,31 @@ @inproceedings{RAK+:VaMoS13
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2012 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{SCC:CISTI12,  
+    author={S. {Sepúlveda} and C. {Cares} and C. {Cachero}},  
+    booktitle={Proc. of the 7th Iberian Conference on Information Systems and Technologies},
+    title= {{Towards a Unified Feature Metamodel: A Systematic Comparison of Feature Languages}},   
+    year= 2012,  
+    pages={1--7}
+}
```
Comment (pmbittner) [inline literature.bib:8453]: make booktitle a variable? CISTI?

Comment (pmbittner) [reply literature.bib:8453]: publisher and address missing

File: literature.bib [hunk 21]
```diff
@@ -8670,7 +8968,7 @@ @inproceedings{CCS+:VaMoS12
 	publisher = ACM,
 	address = NY,
 	tt-tags = {SPL Model Checking,SPL Evolution},
-	tc-tags = {classified by Thomas+Sven,model checking,family-based analysis,family-based specification,annotation-based implementation,design,subsumed by none,CSUR14,PhD15},
+	tc-tags = {classified by Thomas+Sven,model checking,family-based analysis,family-based specification,annotation-basedimplementation,design,subsumed by none,CSUR14,PhD15},
```
Comment (pmbittner) [inline literature.bib:8971]: ?

File: literature.bib [hunk 22]
```diff
@@ -9774,6 +10072,27 @@ @inproceedings{STKS:VaMoS12
...
+}
+
+@article{CB:IST11,
+  title={{A Systematic Review of Evaluation of Variability Management Approaches in Software Product Lines}},
+  author={Chen, Lianping and Babar, Muhammad Ali},
+  journal= IST,
+  volume={53},
+  number={4},
+  pages={344--362},
+  year= 2011
```
Comment (pmbittner) [inline literature.bib:10093]: publisher missing

File: literature.bib [hunk 23]
```diff
@@ -11011,6 +11330,29 @@ @inproceedings{RSTS:VAMOS11
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2010 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+
+@article{APS+:VaMoS10,
+  title={{The CVM Framework-A Prototype Tool for Compositional Variability Management.}},
+  author={Abele, Andreas and Papadopoulos, Yiannis and Servat, David and T{\"o}rngren, Martin and Weber, Matthias},
+  journal= VaMoS,
```
Comment (pmbittner) [inline literature.bib:11337]: Vamos isn't a journal? `@inproceedings`?

File: literature.bib [hunk 24]
```diff
@@ -12678,6 +13020,21 @@ @phdthesis{F09
 	year = 2009
 }
 
+@inproceedings{RZS:ICCS09,
+  author = {Andrei Rimsa and Luis Enrique Z{\'{a}}rate and Mark A. J. Song},
+  title = {{Evaluation of Different {BDD} Libraries to Extract Concepts in {FCA} - Perspectives and Limitations}},
```
Comment (pmbittner) [inline literature.bib:13025]: We can remove duplicate brackets here.

File: literature.bib [hunk 25]
```diff
@@ -18350,6 +18707,20 @@ @inproceedings{MLM96
 	
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 1995 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{PQ:SPE95,
+  author = {Parr, T. J. and Quong, R. W.},
```
Comment (pmbittner) [inline literature.bib:18711]: Can you find the complete names?

File: literature.bib [hunk 26]
```diff
@@ -18350,6 +18707,20 @@ @inproceedings{MLM96
...
+  title = {{ANTLR: A Predicated-LL(k) Parser Generator}},
+  journal = SPE,
+  volume = {25},
+  number = {7},
+  pages = {789-810},
+  keywords = {parsing, compiler, parser generator, predicates, LL(k) parser},
+  doi = {https://doi.org/10.1002/spe.4380250705},
+  year = 1995,
+  publisher={Wiley Online Library}
```
Comment (pmbittner) [inline literature.bib:18720]: There should be a variable for this.

**Review by pmbittner — CHANGES_REQUESTED** [review]

File: literature.bib [hunk 27]
```diff
@@ -33,6 +33,24 @@ @inproceedings{KTSB:ICSE21
...
+  doi = {10.1145/3442391.3442400},
+  articleno = {8},
+  numpages = {10},
+  keywords = {decision models, feature models, Software product lines, orthogonal variability models, variability modeling},
+  location = {Krems, Austria},
+  series = {VaMoS'21},
+  booktitle = VaMoS20,
+ 	publisher = ACM,
+  address = NY,
+  year = 2021
```
[the remark this answers is not in the document]
Comment (tthuem) [reply literature.bib:51]: Yes, we do omit series, as it would be redundant to booktitle including that abbreviation.

File: literature.bib [hunk 28]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
+  year=2019
+}
+
+@inproceedings{BRBA:SPLC19,
+  title={{First International Workshop on Languages for Modelling Variability (MODEVAR 2019)}},
+  author={Benavides, David and Rabiser, Rick and Batory, Don and Acher, Mathieu},
+  booktitle= SPLC,
+  pages={323--323},
+  year= 2019
+}
```
[the remark this answers is not in the document]
Comment (tthuem) [reply literature.bib:1262]: No, this is correct. This is the abstract of the workshop within the proceedings.

File: literature.bib (hunk 21 above)
Comment (tthuem) [reply literature.bib:8971]: This change introduces a misclassification on the website on product-line analyses. To me, this indicates that changes should be considered more critical prior to commit that we can avoid such situtations in reviews.

File: literature.bib [hunk 29]
```diff
@@ -1171,6 +1232,155 @@ @inproceedings{TTS+:VariVolution19
...
 
+@inproceedings{BSE:SPLC19,
+	author = {Beek, Maurice H. ter and Schmid, Klaus and Eichelberger, Holger},
+	title = {{Textual Variability Modeling Languages: An Overview and Considerations}},
+	year = 2019,
+	publisher = ACM,
+	address = NY,
+	booktitle = SPLC,
+	pages = {151–-157},
+	location = {Paris, France}
```
[the remark this answers is not in the document]
Comment (SundermannC) [reply literature.bib:1243]: Does not seem like it. Should I introduce one?

[the remark this answers is not in the document]
Comment (pmbittner) [reply literature.bib:1242]: Would be nice but doesn't have to.

[the remark this answers is not in the document]
Comment (SundermannC) [reply literature.bib]: Done.

File: literature.bib (hunk 16 above)
Comment (SundermannC) [reply literature.bib:4453]: No, but I cannot find the conference whatsoever.

File: literature.bib (hunk 18 above)
Comment (SundermannC) [reply literature.bib:6650]: I think the entry was placed in the wrong year before so I moved it.

File: literature.bib (hunk 16 above)
Comment (pmbittner) [reply literature.bib:4442]: Ok, 🤷‍♂️ .

**Review by pmbittner — APPROVED** [review]
